### PR TITLE
myReact2

### PR DESCRIPTION
**Pull request by mrbarri**

### CODE REVIEW

File: todo-app/src/todo.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import Description from './description'
+import TodoBox from './todoBox'
+import './App.css'
+
+class todo extends Component {
+    render() {
+        return (
+            <div class = 'todoList'>
```
Comment (spirosikmd) [inline todo-app/src/todo.js]: In React we should use `className` instead of `class` due to [jsx](https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx).

File: todo-app/src/todo.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                <div class = 'todoBox'>
+                    <li><TodoBox done={this.props.done}/></li>
+                    </div>
+                    <div class ='Description'>
+                        <li>Description description={this.props.description} /></li>
```
Comment (spirosikmd) [inline todo-app/src/todo.js]: Why did you add this `/>` as part of the text?

File: todo-app/src/todoBox.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    let doneOrNot = 'Not done'
+    if (this.state.clicked) {doneOrNot = "Done"}
+    return doneOrNot
+}
+
+render() {
+    return (
+        <div>
+        <input type = "checkbox"
+        clicked = {this.state.clicked}
```
Comment (spirosikmd) [inline todo-app/src/todoBox.js]: There is no `clicked` attribute for [`input checkbox`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) element. Maybe you wanted to use something else?

File: todo-app/src/todoBox.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        onClick = {this.handleChange}>
```
Comment (spirosikmd) [inline todo-app/src/todoBox.js]: We should use another event handler here and not `onClick` to respond to change. Which one is it?

File: todo-app/src/todoBox.js (hunk 3 above)
Comment (mrbarri) [reply todo-app/src/todoBox.js]: I've checked my code 2 3 times but I think I did mistake I did put some code while we were using last Sunday, reading all code is hard

File: todo-app/src/todo.js (hunk 1 above)
Comment (mrbarri) [reply todo-app/src/todo.js]: @spirosikmd Thanks for feedback . It works here and completely fine in the browser . I don't know where is the problem . so I will read the code again
